src/lsst/rsp/startup/services/labrunner.py

```diff
@@ -822,15 +751,15 @@ def _start(self) -> None:
                 else:
                     self._logger.error(f"Lab process {cmd} failed to run")
                 self._logger.info(f"Waiting for {sleep_interval}s")
-                sleep(sleep_interval)
+                time.sleep(sleep_interval)
             self._logger.debug("Exiting")
             sys.exit(0)
         # Flush open files before exec()
         sys.stdout.flush()
         sys.stderr.flush()
         # In non-debug mode, we don't use a subprocess: we exec the
-        # Jupyter Python process directly.  We use os.execvpe() because we
-        # want the Python in the path (which we currently know to be the
-        # stack Python), we have a list of arguments we just created, and
-        # we want to pass the environment we built up.
+        # Jupyter process directly.  We use os.execvpe() because we
+        # want to use the Python in the path, we have a list of
+        # arguments we just created, and we want to pass the
+        # environment we built up.
```

Comment:
We're no longer using the Python in the path, right?

Reply:
Well, we are.  But now that we're not calling `python` directly, it's the `jupyterhub-singleuser` entrypoint, and it does use the moral equivalent of `$(which python3)`.  I agree that this comment is more confusing than helpful as it stands.